### PR TITLE
Fix compilation error on Windows

### PR DESCRIPTION
**Pull request by bellenot**
Fix compilation error on MSVC after the merge of https://github.com/root-project/root/pull/11756.

### CODE REVIEW

File: tree/ntuple/v7/src/RField.cxx [hunk 1]
```diff
@@ -133,7 +133,7 @@ std::tuple<std::string, std::vector<size_t>> ParseArrayType(std::string_view typ
          return {};
 
       size_t size;
-      if (std::from_chars(std::begin(typeName) + posLBrace + 1, std::begin(typeName) + posRBrace, size).ec !=
+      if (std::from_chars(&(*std::begin(typeName)) + posLBrace + 1, &(*std::begin(typeName)) + posRBrace, size).ec !=
```
Comment (jalopezg-r00t) [inline tree/ntuple/v7/src/RField.cxx:136]:
```suggestion
      if (std::from_chars(typeName.data() + posLBrace + 1, typeName.data() + posRBrace, size).ec !=
```

**Review by jalopezg-r00t — COMMENTED** [review]
Thanks for noticing, @bellenot!  I think that the suggestion should also fix it (and is simpler) :slightly_smiling_face:.

**Comment by bellenot** [comment]
> Thanks for noticing, @bellenot! I think that the suggestion should also fix it (and is simpler) 🙂.

You're right, thanks!

**Review by jalopezg-r00t — APPROVED** [review]

**Comment by jalopezg-r00t** [comment]
> > Thanks for noticing, @bellenot! I think that the suggestion should also fix it (and is simpler) slightly_smiling_face.
> 
> You're right, thanks!

Can we squash the changes in a single commit on merge?  Many thanks!

**Comment by bellenot** [comment]
> > > Thanks for noticing, @bellenot! I think that the suggestion should also fix it (and is simpler) slightly_smiling_face.
> > 
> > 
> > You're right, thanks!
> 
> Can we squash the changes in a single commit on merge? Many thanks!

Indeed!

**Pull request merged**